### PR TITLE
test

### PR DESCRIPTION
**Pull request by kelseiv**
Closes #

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.2/tools/grafana.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ The instructions in this guide require **Grafana Cloud** or **Grafana v7.1+**.
 
 1. [Start InfluxDB OSS {{< current-version >}}](/influxdb/v2.2/install/#start-influxdb).
 2. [Sign up for Grafana Cloud](https://grafana.com/products/cloud/) or
-   [download and install Grafana](https://grafana.com/grafana/download).
+   [download](https://grafana.com/grafana/download) and [install](https://grafana.com/docs/grafana/latest/installation/) Grafana.
```
Comment (kelseiv) [inline content/influxdb/v2.2/tools/grafana.md:27]: typo fix...

File: content/influxdb/v2.2/tools/grafana.md [hunk 2]
```diff
@@ -46,22 +46,21 @@ The instructions in this guide require **Grafana Cloud** or **Grafana v7.1+**.
 With **Flux** selected as the query language in your InfluxDB data source,
 configure your InfluxDB connection:
 
-1. Under **Connection**, enter the following:
+1. Under **HTTP**, enter your [InfluxDB URL](/influxdb/v2.2/reference/urls/). By default, `http://localhost:8086`. Select or enter other HTTP details as needed.
```
Comment (kelseiv) [inline content/influxdb/v2.2/tools/grafana.md:49]: add anotherr

File: content/influxdb/v2.2/tools/grafana.md [hunk 3]
```diff
@@ -46,22 +46,21 @@ The instructions in this guide require **Grafana Cloud** or **Grafana v7.1+**.
...
 
-    - **URL**: Your [InfluxDB URL](/influxdb/v2.2/reference/urls/).
+{{% note %}}
+If **Server access** is selected, this URL must be accessible from the Grafana server. If **Browser access** is selected, this URL must be accessible from a browser.
```
Comment (kelseiv) [inline content/influxdb/v2.2/tools/grafana.md:52]: jdsf;sdj;fjklsj

**Review by kelseiv — COMMENTED** [review]
Hi Kelly, a couple minor edits

**Review by lwandzura — APPROVED** [review]
LGTM

**Comment by kelseiv** [comment]
closing per https://github.com/influxdata/docs-v2/pull/4107

**Pull request closed without merging**